src/services/hospitalInfo/hospitalInfoService.ts

```diff
@@ -67,6 +67,14 @@ class HospitalInfoService {
   }
 }
 
+function isHospitalOpen(hospitalInfo: HospitalInfo | undefined) {
```

Comment:
I would move this function into the service.

then change MockHospitalInfoService to
```
class MockHospitalInfoService extends HospitalInfoService
```

Do forget to write a test for this new function :-)